### ISSUE TITLE
Clustered TerminatedServerTest failing on windows

### ISSUE THREAD

**Issue by rkavanap** [ehcache/ehcache3#1217]
The following test is failing on windows with the latest ehcache on trunk..

java.lang.IllegalStateException: Active process did not appear
	at org.terracotta.testing.master.SynchronousProcessControl.internalWaitForActive(SynchronousProcessControl.java:256)
	at org.terracotta.testing.master.SynchronousProcessControl.waitForActive(SynchronousProcessControl.java:177)
	at org.terracotta.testing.rules.BasicExternalCluster$3.waitForActive(BasicExternalCluster.java:197)
	at org.ehcache.clustered.TerminatedServerTest.waitForActive(TerminatedServerTest.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:109)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:377)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
	at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

**Comment by gibsong** [comment]
I am no longer getting the error referenced above on my windows machine.  Now when I run "./gradlew test clean" and enable logging in the gradle.build file, I see the following in the logs:

org.ehcache.clustered.TerminatedServerTest > testTerminationBeforeCacheManagerClose STANDARD_OUT                                                                                                        
[testTerminationBeforeCacheManagerClose(org.ehcache.clustered.TerminatedServerTest)][ConfigBuilder] Added testServer0                                                                               
[testTerminationBeforeCacheManagerClose(org.ehcache.clustered.TerminatedServerTest)][ProcessControl] >>> addServerAndStart                                                                                       
                                                                                                                                                                                                       
org.ehcache.clustered.TerminatedServerTest > testTerminationBeforeCacheManagerClose STANDARD_ERROR                                                                                                      
    [testTerminationBeforeCacheManagerClose(org.ehcache.clustered.TerminatedServerTest)][testServer0] 'C:\Users\ggib\dev\ehcache3_dev\ehcache3\clustered\integration-test\build\cluster\testTerminationBeforeCacheManagerClose' is not recognized as an internal or external command,                                                                                                                                        
[testTerminationBeforeCacheManagerClose(org.ehcache.clustered.TerminatedServerTest)][testServer0] operable program or batch file.                                                                                                                                                                                                                                                                       
[ConnectionEstablisher-0-1382641315] WARN class.com.tc.net.protocol.transport.ClientMessageTransport - ConnectionID(4.e11e3885ee3f49f9bfef5d132808db10.8a35f41c-974a-40b2-8bbf-fa59898d2142-155c0f614c1.USER)[]: 
Connection refused: no further information                                                                                                                                                                           
[ConnectionEstablisher-0-1621991228] WARN class.com.tc.net.protocol.transport.ClientMessageTransport - ConnectionID(5.e11e3885ee3f49f9bfef5d132808db10.8a35f41c-974a-40b2-8bbf-fa59898d2142-155c0f614c1.USER)[]: 
Connection refused: no further information                                                                                                                                                             

......

Eventually it kept printing the following log entries:

 [ConnectionEstablisher-0-366477705] WARN class.com.tc.net.protocol.transport.ClientMessageTransport - ConnectionID(4.7e5340857e7c489b84de5304b45cd014.71ffec15-9db9-4784-b34a-bdabf3f904c1-155bdc7239a.USER)[]: Connection refused: no further information [lossy interval: 10000ms]

and

[ConnectionEstablisher-0-40841753] WARN class.com.tc.net.protocol.transport.ClientMessageTransport - ConnectionID(5.7e5340857e7c489b84de5304b45cd014.71ffec15-9db9-4784-b34a-bdabf3f904c1-155bdc7239a.USER)[]: Reconnect attempt 140 of unlimited reconnect tries to localhost:32732, timeout=10000


when I killed the test via ctrl c, I got the following error:
Connection refused: no further information [lossy interval: 10000ms]
Unexpected exception thrown.
org.gradle.internal.remote.internal.MessageIOException: Could not write '/127.0.0.1:50023'.
        at org.gradle.internal.remote.internal.inet.SocketConnection.flush(SocketConnection.java:121)
        at org.gradle.internal.remote.internal.hub.MessageHub$ConnectionDispatch.run(MessageHub.java:300)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
        at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: An existing connection was forcibly closed by the remote host
        at sun.nio.ch.SocketDispatcher.write0(Native Method)
        at sun.nio.ch.SocketDispatcher.write(SocketDispatcher.java:51)
        at sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:93)
        at sun.nio.ch.IOUtil.write(IOUtil.java:51)
        at sun.nio.ch.SocketChannelImpl.write(SocketChannelImpl.java:471)
        at org.gradle.internal.remote.internal.inet.SocketConnection$SocketOutputStream.writeWithNonBlockingRetry(SocketConnection.java:259)
        at org.gradle.internal.remote.internal.inet.SocketConnection$SocketOutputStream.writeBufferToChannel(SocketConnection.java:247)
        at org.gradle.internal.remote.internal.inet.SocketConnection$SocketOutputStream.flush(SocketConnection.java:241)
        at org.gradle.internal.remote.internal.inet.SocketConnection.flush(SocketConnection.java:119)
        ... 6 more
